rhel8/profiles/stig_gui.profile

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+metadata:
+    version: V1R1
+    SMEs:
+        - carlosmmatos
+
+reference: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux
+
+title: 'DISA STIG with GUI for Red Hat Enterprise Linux 8'
+
+description: |-
+    This profile contains configuration checks that align to the
+    DISA STIG with GUI for Red Hat Enterprise Linux 8.
```

Comment:
I think we have missed the versioning for RHEL8 STIG profile. We should have `V1R1` stated in this line.
```suggestion
    DISA STIG with GUI for Red Hat Enterprise Linux 8 V1R1.
```

Would you care to change in the original stig.profile as well? On the other hand V1R2 is about to be released.

Reply:
Roger that.